OpenAPI/openapidoc.json

```diff
@@ -1473,7 +1473,7 @@
           "Path": {
             "type": "string",
             "description": "File localization",
-            "example": "\\files\\T001\\Files\\tmp_loadfile\\loadfile.dat"

```

Comment:
this openapidoc.json is the same as in our import repo?

if it is then we need to remeber to upadate this after updates in bitbucket

Reply:
They are a little bit different, but should probably be the same. Yes - I'll update this file after Igor's changes to original repo.